### PR TITLE
Seminar

### PR DESCRIPTION
**Pull request by YussufOB**
Kindly help to review my. The following were done:

1. Construct HTML files for home and about pages using appropriate semantic.
2. Create style to support the U.I of the pages using CSS.
4. Add the 'speakers' section dynamically using javascript and some other features from the requirements.
5. Make changes required by the reviewer.

Loom Link
https://www.loom.com/share/b3fbdb57749f41e786f97740d492b5cc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to all feature person and the brands that their logo was used.
+- Inspiration comes from my interest in technical analysis and support for the Cotton Candy work.
+
```
Comment (gtekle) [inline README.md]: - [ ] You should give credit to [Cindy Shin in Behance](https://www.behance.net/adagio07) for the Original design idea, as stated in the project requirement. 
> The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @YussufOB :wave:, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
dog-pug


## Suggested changes
_Check the comments under the review._

- [ ] You should record a video presentation of your project according to project requirement specification. 
> Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. 
The content of the video should include
    - A description of the project.
    - A demo of the project features (different pages and different screen sizes).
    - You should also highlight some interesting piece of code or something you built that you are very proud of.

- [ ] You should add a footer for the About page as per the design. Please take a look at these snapshots from the design. 
#### Mobile Version (About page footer)
about-page-footer-mob
#### Desktop Version (About page footer)
about-page-footer-desk

- [ ] You should add the `See the past Summits` section in the About page. Please take a look at the following snapshots from the design.
  #### Mobile V
about-page-past-summits-mob
ersion
  #### Desktop Version
about-page-past-summits-desk

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 2]
```diff
@@ -1 +1,65 @@
...
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Live Demo Links
+
+[Live Demo Link](https://yussufob.github.io/Seminar--Capstone-/)
+[Live Demo Link](https://www.loom.com/share/b3fbdb57749f41e786f97740d492b5cc)
```
Comment (DeliceLydia) [inline README.md:17]: ## Optional
- [ ] I suggest that you show that one link is a live demo then another one is a video demo to be clear you can simply do:

## Live demo
---

## Video demo
---

**Review by DeliceLydia — APPROVED** [review]
## Hi @YussufOB, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


### Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**